docs/select/README.md

```diff
@@ -90,9 +88,21 @@ Stats details bytesProcessed:
 25786743
 ```
 
+For more detailed SELECT SQL reference refer [here](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-select.html)
+
 ## 5. Explore Further
 - [Use `mc` with Minio Server](https://docs.minio.io/docs/minio-client-quickstart-guide)
+- [Use `mc sql` with Minio Server](https://docs.minio.io/docs/minio-client-complete-guide.html#sql)
 - [Use `minio-go` SDK with Minio Server](https://docs.minio.io/docs/golang-client-quickstart-guide)
 - [Use `aws-cli` with Minio Server](https://docs.minio.io/docs/aws-cli-with-minio)
 - [Use `s3cmd` with Minio Server](https://docs.minio.io/docs/s3cmd-with-minio)
 - [The Minio documentation website](https://docs.minio.io)
+
+## 6. Implementation Status
+- Full AWS S3 [SELECT SQL](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-select.html) syntax is supported.
+- All [operators](https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-operators.html) are supported.
+- All aggregation, conditional, type-conversion and strings functions are supported.
```

Comment:
Is this string or strings?

Reply:
It's called strings .. it's a method `STRINGS`

Reply:
If we are going to be pedantic about it it is "string functions" - see https://docs.aws.amazon.com/AmazonS3/latest/dev/s3-glacier-select-sql-reference-string.html

Reply:
updated the PR @eco-minio @donatello PTAL